### ISSUE TITLE
Hoodle closes when trying to open incorrect file

### ISSUE THREAD

**Issue by ivanperez-keera** [wavewave/hoodle-core#2]
I just tried to open  a xournal file with hoodle and it closes without warning or any information. The error.log file says:

2012-12-07 14:31:10 UTC : TextException Cannot decode byte '\x8b': Data.Text.Encoding.decodeUtf8: Invalid UTF-8 stream

**Comment by wavewave** [comment]
Yes, this happens. I will make hoodle at least not crash but just give error message box for this. Thanks.
